sys-libs/gpm/gpm-1.20.7-r3.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# emacs support disabled due to #99533 #335900
+
+EAPI=7
+
+inherit systemd toolchain-funcs autotools multilib-minimal usr-ldscript
+
+DESCRIPTION="Console-based mouse driver"
+HOMEPAGE="https://www.nico.schottelius.org/software/gpm/"
+SRC_URI="https://www.nico.schottelius.org/software/${PN}/archives/${P}.tar.lzma
+	mirror://gentoo/${P}-docs.patch.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="selinux static-libs"
+
+RDEPEND=">=sys-libs/ncurses-5.9-r3:0=[${MULTILIB_USEDEP}]
+	selinux? ( sec-policy/selinux-gpm )"
+DEPEND=">=sys-libs/ncurses-5.9-r3:0=[${MULTILIB_USEDEP}]
+	app-arch/xz-utils
+	sys-apps/texinfo
+	virtual/yacc"
```

Comment:
You should probably move these packages to BDEPEND:
```
app-arch/xz-utils
sys-apps/texinfo
virtual/yacc
```

Reply:
Done, moved.